cmd/aspect/root/BUILD.bazel

```diff
@@ -13,10 +13,11 @@ go_library(
         "//cmd/aspect/clean",
         "//cmd/aspect/docs",
         "//cmd/aspect/info",
+        "//cmd/aspect/query",
         "//cmd/aspect/root/flags",
         "//cmd/aspect/test",
         "//cmd/aspect/version",
-        "//docs/help/topics",
+        "//docs/help/topics",  # keep
```

Comment:
I think this `keep` can be removed now - the issue I was working around has been resolved